### PR TITLE
Add unit tests for UnderFileSystemBlockStore

### PR DESCRIPTION
**Pull request by YangchenYe323**
### What changes are proposed in this pull request?
I added some tests for `UnderFileSystemBlockStore`

### Why are the changes needed?
To test the correctness of `UnderFileSystemBlockStore#createBlockReader` and some edge cases.

Improve test coverage for the class(30% -> 80%)

### Does this PR introduce any user facing changes?
No

### CODE REVIEW

File: core/server/worker/src/test/java/alluxio/worker/block/UnderFileSystemBlockStoreTest.java [hunk 1]
```diff
@@ -60,6 +98,106 @@ public void releaseAccess() throws Exception {
...
+  @Test
+  public void createBlockReader() throws Exception {
+    // flush some data to test file
+    byte[] data = new byte[(int) TEST_BLOCK_SIZE];
+    for (int i = 0; i < TEST_BLOCK_SIZE; ++i) {
+      data[i] = 1;
+    }
+    BufferedOutputStream out = new BufferedOutputStream(new FileOutputStream(mTempFile));
+    out.write(data, 0, data.length);
+    out.flush();
```
[the remark this answers is not in the document]
Comment (YangchenYe323) [reply core/server/worker/src/test/java/alluxio/worker/block/UnderFileSystemBlockStoreTest.java]: Fixed.

File: core/server/worker/src/test/java/alluxio/worker/block/UnderFileSystemBlockStoreTest.java [hunk 2]
```diff
@@ -60,6 +98,106 @@ public void releaseAccess() throws Exception {
...
+        .setUfsPath(mTempFilePath)
+        .build();
+
+    BlockReader reader = blockStore
+        .createBlockReader(sessionId, BLOCK_ID, 0, false, options);
+
+    assertArrayEquals(data, reader.read(0, TEST_BLOCK_SIZE).array());
+
+    blockStore.close(sessionId, BLOCK_ID);
+    blockStore.releaseAccess(sessionId, BLOCK_ID);
```
[the remark this answers is not in the document]
Comment (YangchenYe323) [reply core/server/worker/src/test/java/alluxio/worker/block/UnderFileSystemBlockStoreTest.java]: Fixed.

**Comment by YangchenYe323** [comment]
@jiacheliu3 Could we merge this? Thanks!

**Comment by jiacheliu3** [comment]
alluxio-bot, merge this please
